pwndbg/commands/argv.py

```diff
@@ -13,43 +15,34 @@
 import pwndbg.typeinfo
 
 
-@pwndbg.commands.Command
+@pwndbg.commands.ArgparsedCommand(argparse.ArgumentParser(description='Prints out the number of arguments.'))
```

Comment:
I'd greatly prefer to break down everything onto its own line, **or** to do pass-through on `kwargs`